### PR TITLE
Login feature

### PR DESCRIPTION
**Pull request by enageswararao**
Feature is completed please revew

### CODE REVIEW

File: src/main/java/com/example/pages/DashBoardPage.java [hunk 1]
```diff
@@ -1,5 +1,46 @@
...
 
+import java.sql.Driver;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
 public class DashBoardPage {
 
+	WebDriver driver;
```
Comment (RajeshThodetiQA) [inline src/main/java/com/example/pages/DashBoardPage.java:11]: Mark this driver as webdriver.

Comment (RajeshThodetiQA) [reply src/main/java/com/example/pages/DashBoardPage.java:11]: TEST

File: src/test/java/com/example/DashBoradTest.java [hunk 2]
```diff
@@ -1,5 +1,68 @@
...
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import com.example.pages.DashBoardPage;
+import com.example.pages.LoginPage;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
 public class DashBoradTest {
+
```
Comment (enageswararao) [inline src/test/java/com/example/DashBoradTest.java:17]: Remove space

Comment (enageswararao) [reply src/test/java/com/example/DashBoradTest.java:17]: trest

**Pull request merged**